tests/providers/gcp/gcp_managed_relational_db_test.py

```diff
@@ -34,11 +35,26 @@ class GcpManagedRelationalDbSpecTestCase(unittest.TestCase):
   pass
 
 
-class GceManagedRelationalDbFlagsTestCase(unittest.TestCase):
+class GcpManagedRelationalDbFlagsTestCase(unittest.TestCase):
   pass
 
 
-class GceManagedRelationalDbTestCase(unittest.TestCase):
+class GcpManagedRelationalDbTestCase(unittest.TestCase):
+
+  def createSpecDict(self):
+    vm_spec = virtual_machine.BaseVmSpec('NAME',
+                                         **{'machine_type': 'db-n1-standard-1'})
+    disk_spec = disk.BaseDiskSpec('NAME',
+                                  **{'disk_size': 50})
+    return {
+        'database': MYSQL,
+        'database_version': '5.6',
```

Comment:
Along the same line as my comments earlier, I suggest you try writing a test that does not specify the version to ensure that your code will return the correct default version for the given database. Again, a TODO is ok for now.